### ISSUE TITLE
Loss of train and validation every epoch

### ISSUE THREAD

**Issue by lyu124** [saic-mdal/lama#201]
Hello, i want to get the loss of train and validation after every epoch, is that possible? How can i realize it? thank you so much

**Comment by windj007** [comment]
The code based on pytorch-lightning - there is a [hook](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#train-epoch-level-operations) for training and a similar hook for validation. You can save the metrics you need by implementing these hooks
